test/run_all.py

```diff
@@ -57,6 +57,11 @@
     test_files.remove('tst_cdl.py');
     sys.stdout.write('not running tst_cdl.py ...\n')
 
+# Don't run computationally intensive test
+if os.getenv('MEMORY_LEAK_TEST'):
```

Comment:
I think we want `if not os.getenv('MEMORY_LEAK_TEST'):` so the test doesn't run unless the env var is set?